### PR TITLE
Add Inventory Class and ViewInventoryCommand

### PR DESCRIPTION
**Pull request by bbryant824**
Add Inventory Class to manage the ingredients in stock.
Add ViewInventoryCommand to list down the quantity of each Ingredient in stock by its name and unique ID.

### CODE REVIEW

File: src/main/java/seedu/address/model/product/Inventory.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            Ingredient ingredient = ingredientCatalogue.getIngredientById(entry.getKey());
+            System.out.printf("ID: %d | %s: %d units%n",
+                    ingredient.getProductId(), ingredient.getName(), entry.getValue());
+        }
+    }
+
+    public String getFormattedStockLevels() {
+        StringBuilder sb = new StringBuilder();
+        for (Map.Entry<Integer, Integer> entry : stockLevels.entrySet()) {
+            Ingredient ingredient = ingredientCatalogue.getIngredientById(entry.getKey());
```
Comment (calebchongsj) [inline src/main/java/seedu/address/model/product/Inventory.java:84]: Great use of string builder to return the ingredient stock levels.

**Review by calebchongsj — APPROVED** [review]
Looks good to me. Good job, Bowen!

**Review by ryenl — COMMENTED** [review]
LGTM!

**Comment by yikyak02** [comment]
good that you made a new Inventory class in line with proper OOP principles, keep up the good work!

**Pull request merged**